extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -95,6 +113,49 @@ public static function onRawPageViewBeforeOutput( \RawAction $rawAction, &$text
 		return true;
 	}
 
+	/*
+	 * Adds review status item to top nav tabs in Monobook skin.
+	 * This is an entrypoint for checking review status and submitting changes for review/
+	 * This is attached to the MediaWiki 'SkinTemplateNavigation' hook.
+	 * @param SkinTemplate $skin
+	 * @param array $links Navigation links
+	 * @return bool true
+	 */
+	public static function onSkinTemplateNavigation( $skin, &$links ) {
+		global $wgCityId, $wgTitle;
+
+		if ( $skin->getSkinName() !== 'monobook' ||
+			!self::userCanEditJsPage()
+		) {
+			return true;
+		}
+
+		/* Get page status */
+		$pageStatus = \F::app()->sendRequest(
+			'ContentReviewApiController',
+			'getPageStatus',
+			[
+				'wikiId' => $wgCityId,
+				'pageId' => $wgTitle->getArticleID(),
+			],
+			true
+		)->getData();
+
+		/* Determine review status */
+		$latestRevisionId = (int)$wgTitle->getLatestRevID();
```

Comment:
`getLatestRevID` always returns an integer so you don't need the additional type casting here.